cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/02-costs.html

```diff
@@ -84,52 +104,63 @@ <h3 class="u-border-bottom">Indirect costs</h3>
                 })
             }}
 
-            {{  
-                text_item.render({  
-                    'label': 'Estimated indirect costs',    
-                    'value': '$99,999', 
-                    'type': 'subtotal', 
-                    'status': 'warning',    
-                    'data_attribute': 'data-financial-item="total_indirectCosts"'   
-                })  
-            }}
-            
-            {{
-                input.render({
-                    'label': 'Additional financial obligations',
-                    'helper_text': 'Expenses not related to school and not eligible for financial aid, like car payments, credit card payments, helping family with bills, etc. Ask the financial aid office if you\'re not sure if an expense qualifies for financial aid.',
-                    'id': 'costs__otherIndirect',
-                    'value': '0',
-                    'data_attribute': 'data-financial-item="otherCost_additional"'
-                })
-            }}
-
-            {{
-                text_item.render({
-                    'label': 'Your estimated costs this year',
-                    'value': '$99,999',
-                    'type': 'total',
-                    'status': 'warning',
-                    'data_attribute': 'data-financial-item="total_costs"'
-                })
-            }}
-        
             {{ notification.render(
-                'money-tips',
+                'caution',
                 true,
-                'Money saving tips',
-                '<ul>
-                        <li> Finish school on time. This typically requires earning 30 credits a year for an associate’s or bachelor’s. Work with your academic adviser to choose classes that add up to a degree. Graduating on time (or early) will help minimize your costs, maximize your aid eligibility, and get you into a full-time job sooner.</li>
-                        <li>Find out if this school offers block tuition (also known as flat rate tuition) &mdash; when it costs the same to take anywhere from 12 to 18 credits, for example.</li>
-                        <li data-state-based-visibility="costs-question-yes">If your costs will be higher than the school predicts, contact the financial aid office. You may be eligible for more aid. </li>
-                </ul>' | safe
+                '',
+                'If your actual costs will be higher than the estimates from the school, talk to the financial aid office. You may be able to appeal for more aid.' | safe
             ) }}
+            <ul>
+                <li>You may also qualify for additional aid if you will be paying for health insurance, childcare, a personal computer, disability support, or study abroad.</li>
+                <li>Just because you qualify for more aid doesn’t mean you will receive it. And remember, financial aid includes student loans. More financial aid can mean an offer to borrow more money.</li>
+            </ul>
```

Comment:
I wonder if the above content could / should be in a notification block? That seems to be the method we generally use when interspersing guidance with form fields...

Reply:
Sure, we can ask the team what they think. Easy change to make, if we want to.

Reply:
Maybe we can add this to the general hierarchy updates work